packages/amplify-graphql-transformer-interfaces/API.md

```diff
@@ -146,6 +146,16 @@ export interface AssetProvider {
     provide: (scope: Construct, name: string, props: AssetProps) => S3Asset;
 }
 
+// @public (undocumented)
+export interface BranchIdentifier {
```

Comment:
See note above about naming. While the internal interfaces don't strictly have to correspond to the external interfaces, in cases where the serve the same purpose with the same data, it reduces our confusion to keep the names the same.